modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -739,4 +739,80 @@ definition {
 		}
 	}
 
+	@description = "LPS-178783 - Verify a standalone action can be triggered manually to send an email"
+	@priority = 5
+	test CanSendEmailViaActionStandalone {
+		property custom.properties = "feature.flag.LPS-173894=true";
+		property portal.acceptance = "true";
+		property test.name.skip.portal.instance = "ObjectNotifications.CanSendEmailViaActionStandalone";
+		property test.smtp.server.enabled = "true";
+
+		task ("Given an Email Notification Template") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				emailBody = "Email Body",
+				entryTo = "test@liferay.com",
+				fromAddress = "test@liferay.com",
+				fromName = "Test Test",
+				name = "Notification Template",
+				notificationType = "Email",
+				subject = "Subject");
+
+			PortletEntry.save();
+		}
+
+		task ("And Custom Object with an Action (Standalone)") {
```

Comment:
Can you split this into two tasks? One task to create the custom object and entry, one task to add a standalone action. You can use your best judgement for the wording of the steps.